src/server.mjs

```diff
@@ -1154,13 +1303,27 @@ function getBedRegions(bedFile, dataPath) {
     }
     bed_info["chunk"].push(chunk);
 
+  });
 
+  // download chunks if bedfile is an url
+  if (isURL) {
+    await retrieveChunks(bedFile, bed_info["chunk"]);
```

Comment:
Does it make sense to retrieve all the BED chunk data up front? We currently seem to limit the BED file size to a bit over 4 kilobytes, but really we're going to want to support cases like a BED file of 20,000 different genes. So as soon as we raise the BED file size limit we're going to want to come up with a way to lazily fetch just the chunks people actually go to.